reana_client/cli/files.py

```diff
@@ -534,6 +534,59 @@ def move_files(ctx, source, target, workflow, access_token):  # noqa: D301
         sys.exit(1)
 
 
+@files_group.command("prune")
+@check_connection
+@add_workflow_option
+@add_access_token_options
+@click.option(
+    "--include-inputs",
+    "include_inputs",
+    is_flag=True,
+    help="Also prune the input files associated with the workflow.",
+)
+@click.option(
+    "--include-outputs",
+    "include_outputs",
+    is_flag=True,
+    help="Also prune the output files associated with the workflow.",
+)
+@click.pass_context
+def prune_files(ctx, workflow, access_token, include_inputs, include_outputs):
+    r"""
```

Comment:
At first I added it because flake8 would mark the absence of the `r` as an error, but I have just realized I should include `# noqa: D301` to ignore that warning, similarly to how the other functions of the file do. Thanks for pointing that out!